Week1/homework/style.css

```diff
@@ -1 +1,36 @@
-/* add your styling here */
\ No newline at end of file
+html {
```

Comment:
The styling here is a little bit lacking.
Try exploring this a bit more, you could use `grid` to better layout your list of books.
Also try test your site on different size screens - users will view your sites on mobile devices too so make sure it looks good on all sizes. See if you can try improve this.